themes/default/content/learn/building-with-pulumi/stack-references/index.md

```diff
@@ -127,6 +137,47 @@ export then grabs the `url` output from the other stack.
 
 {{% /choosable %}}
 
+{{% choosable language go %}}
+
+```go
+package main
+
+import (
+	"fmt"
+
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi/config"
+)
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+		cfg := config.New(ctx, "")
+		stack := ctx.Stack()
+		org := cfg.Require("org")
+
+		stackRef, err := pulumi.NewStackReference(ctx, fmt.Sprintf("%v/my-first-app/%v", org, stack), nil)
+		if err != nil {
+			return err
+		}
+
+		ctx.Export("shopUrl", stackRef.GetOutput(pulumi.String("url")))
+		return nil
+	})
+}
+```
+
+You'll also need to run `go mod tidy` in the `my-second-app` directory.
+
+The `org` environment variable is new, as is the `stackRef` declaration. That
```

Comment:
Should this be configuration?

```suggestion
The `org` configuration variable is new, as is the `stackRef` declaration. That
```

We call this an "environment variable" in a few places, looks like.

Reply:
Changed all instances in c54cc22